test/custom-task-ctrls/wait-task/pkg/reconciler/reconciler.go

```diff
@@ -127,9 +127,30 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, r *v1alpha1.Run) kreconc
 		logger.Infof("The Custom Task Run %v timed out", r.GetName())
 		r.Status.CompletionTime = &metav1.Time{Time: c.Clock.Now()}
 		r.Status.MarkRunFailed("TimedOut", WaitTaskCancelledByRunTimeoutMsg)
+
+		// Retry if retries > len(run.Status.RetriesStatus), retry it
```

Comment:
Wondering if it makes sense to be more descriptive here saying "If the current RetriesStatus hasn't reached the retries limit". Currently the comment looks quite duplicated with the code in #L132.

Reply:
Yep, it can make sense. Updated.